pe-parser-library/src/parse.cpp

```diff
@@ -2434,6 +2539,13 @@ parsed_pe *ParsePEFromBuffer(bounded_buffer *buffer) {
     return nullptr;
   }
 
+  if (!getDebugDir(p)) {
```

Comment:
Is the debug directory always present? We might want to handle the case where it's missing entirely (instead of malformed) separately as a non-error case.

Reply:
Inside the function we return `true` if `debugDir.Size == 0`. I think that should handle those cases?

Reply:
Missed that, thanks!